Jenkinsfile

```diff
@@ -0,0 +1,260 @@
+#!groovy
+
+node {
+    def TIDB_TEST_BRANCH = "master"
+    def TIDB_BRANCH = "master"
```

Comment:
can we pass these from command flags so we can use the same code for master and rc2.2.